docs/articles/actors/routers.md

```diff
@@ -96,6 +96,27 @@ Routers are implemented as actors, so a router is supervised by it's parent, and
 
 By default, pool routers use a custom strategy that only returns `Escalate` for all exceptions, the router supervising the failing worker will then escalate to it's own parent, if the parent of the router decides to restart the router, all the pool workers will also be recreated as a result of this.
 
+## Sending messages to actors behind router
+
+As routers are actors, we send over messages to actors they manage the same way we communicate to other actors via `ActorRef`:
+
+```cs
+var router = system.ActorOf(Props.Create<Worker>().WithRouter(new RoundRobinPool(3)), "some-pool");
+foreach (var element in Enumerable.Range(0, 6))
+{
+  router.Tell(new Message(element));
+}
+```
+
+Router will send over messages according to its strategy.
+In this particular case, we have `RoundRobinPool` configuration that will:
+* create 3 instances (because it's Pool strategy - see above) of `Worker` actor 
+* enqueue<sup>1</sup> total of 6 messages of type `Message` to actors **mailboxes** it manages according to its RoundRobin routing strategy
```

Comment:
..... **Tell** (not enqueue)  **a** total of 6 messages ....... to its `routees` according to its RoundRobin routing strategy.

Dont use words like Enqueue and Mailboxes here. Because its a detail of the IActorRef implementation. Routee's are nothing more then IActorRef's. Whether or not messages are enqueued or not, is a detail of the IActorRef implementation you should not mention here, because the Router Actor has no knowledge of these details.

Same goes for mailboxes. What if the IActoRef is actually a RemoteActorRef because the routees being used are actually remote actors ? A lot more stuff is happening then. Again, its an implementation detail the Router Actor has no knowledge about so don't describe that here.